### PR TITLE
[p5.js hi] events references

### PR DESCRIPTION
**Pull request by Surbhi-Pittie**

### CODE REVIEW

File: src/content/reference/hi/p5/doubleClicked.mdx [hunk 1]
```diff
@@ -1,80 +1,80 @@
...
-module: Events
-submodule: Mouse
+module: आयोजन
+submodule: माउस
 file: src/events/mouse.js
 description: >
-  <p>A function that's called once when a mouse button is clicked twice
-  quickly.</p>
+  <p>एक फ़ंक्शन जिसे माउस बटन पर दो बार क्लिक करने पर एक बार कॉल किया जाता है
+  जल्दी.</p>
```
Comment (Divyansh013) [inline src/content/reference/hi/p5/doubleClicked.mdx:8]: जल्दी कॉल किया जाता है  should be write imo

File: src/content/reference/hi/p5/doubleClicked.mdx [hunk 2]
```diff
@@ -1,80 +1,80 @@
...
 
-  <p>Declaring the function <code>doubleClicked()</code> sets a code block to
-  run
+  <p>फ़ंक्शन घोषित करना <code>doubleClicked()</code> एक कोड ब्लॉक सेट करता है
+  दौड़ना
```
Comment (Divyansh013) [inline src/content/reference/hi/p5/doubleClicked.mdx:11]: code 'chalne' would be write here

File: src/content/reference/hi/p5/keyPressed.mdx [hunk 3]
```diff
@@ -1,72 +1,72 @@
...
-  <code>keyPressed()</code> is called by p5.js:</p>
+  <code>keyPressed()</code> को p5.js द्वारा कॉल किया जाता है:</p>
 
   <code>
 
-  function keyPressed() {
-    if (key === 'c') {
-      // Code to run.
+  फ़ंक्शन keyPressed() {
+    यदि (कुंजी === 'सी') {
```
Comment (Divyansh013) [inline src/content/reference/hi/p5/keyPressed.mdx:33]: should be 'c' only

File: src/content/reference/hi/p5/keyReleased.mdx [hunk 4]
```diff
@@ -1,70 +1,70 @@
...
+  एक बार
 
-  automatically when the user releases any key:</p>
+  जब उपयोगकर्ता कोई कुंजी जारी करता है तो स्वचालित रूप से:</p>
 
   <code>
 
-  function keyReleased() {
-    // Code to run.
+  फ़ंक्शन कुंजी जारी() {
```
Comment (Divyansh013) [inline src/content/reference/hi/p5/keyReleased.mdx:16]: don't change

File: src/content/reference/hi/p5/keyTyped.mdx [hunk 5]
```diff
@@ -1,86 +1,86 @@
...
+  <code>नियंत्रण</code>, और तीर कुंजियाँ
 
-  will be ignored:</p>
+  नजरअंदाज कर दिया जाएगा:</p>
 
   <code>
 
-  function keyTyped() {
-    // Code to run.
+  फ़ंक्शन कुंजी टाइप किया गया() {
```
Comment (Divyansh013) [inline src/content/reference/hi/p5/keyTyped.mdx:22]: keep function as it is

**Review by Divyansh013 — CHANGES_REQUESTED** [review]

**Review by Divyansh013 — APPROVED** [review]

**Comment by Divyansh013** [comment]
@TanviKumar PTAL

**Review by TanviKumar — APPROVED** [review]

**Comment by Divyansh013** [comment]
@Qianqianye @limzykenneth we can merge this PR

**Comment by Qianqianye** [comment]
Looks great, thank you all!

**Pull request merged**